app/src/main/java/com/example/mycalculator/mvp/ActivityView.java

```diff
@@ -0,0 +1,4 @@
+package com.example.mycalculator.mvp;
+
+public class ActivityView {
```

Comment:
why this class is empty?

Reply:
After the meeting i had with Lucho and Mau, they told me about the weak references, and how to use the ActivityView, more efficiently. Thx!!